src/components/PublishContainer/PublishedScreen/PublishedScreen.tsx

```diff
@@ -128,7 +128,7 @@ export const PublishedScreen: FunctionComponent<PublishScreen> = ({
                 const size = prettyBytes(getFileSize(JSON.stringify(doc.wrappedDocument)));
                 return (
                   <div key={index} className="flex items-center">
-                    <div className="font-bold text-lightgrey-dark">{doc.fileName}</div>
+                    <div className="font-bold text-lightgrey-dark">{`${doc.fileName}.tt`}</div>
```

Comment:
```suggestion
                    <div className="font-bold text-lightgrey-dark">{doc.fileName}.tt</div>
```

Just another way to go about it... fine either way tbh

Reply:
ok